Add-ons/Starter-Kit-TEMP/Widgets/index.md

```diff
@@ -71,6 +71,15 @@ The example below uses, a ***Pre Heading***, a ***Heading***, a ***Text*** and t
 
 The Gallery Widget allowes you to display multiple images in a gallery view, meaning that the images will be alligned in rows. 
 This also contians the feature to click on images to zoom in on the images(this feature can also be disabled).
+You will be able to change the set up of your gallery for example the size of the images and the background color of the whole container. 
```

Comment:
```suggestion
You will be able to change the setup of your gallery. This could be the size of the images or the background color of the whole container. 
```